power_spec.adoc

```diff
@@ -0,0 +1,161 @@
+wish to submit power must follow this spec. This power specification covers
+wall-powered systems only. This power spec is currently only applicable to the
+Inference benchmark. We expect future versions will support additional features
+to enable its application for the MLPerf Training Suite.
+
+== Power management
+
+The goal of this benchmark is to mimic real-world usage scenarios as much
+as possible, and enable demonstrating of the benefits of realistic power
+management. Therefore, we require any power management system to:
+
+* Be qualified for use appropriate for the
+submission type (e.g., a generally available system must use
+software/firmware qualified for general availability and shipping
+with the platform)
+* Include no benchmark or benchmarking-specific modifications
+* Changes in power management behavior must not have manual intervention or
+be aware of the benchmark
+
+== Wall-Powered System Under Test
+
+The System under Test (SUT) includes any component that contributes to the
+measured performance of the MLPerf benchmark. For example, an
+accelerator based system should also include the host node,
+networking/switches for distributed systems, number and type of cooling systems,
+and hard disks. If the system uses shared resources with multiple servers, the
+shared resources must be included in the SUT.
+
+The cooling method must be described (e.g., air-cooling, liquid cooling).
+
+Figure 1 below illustrates details of the MLPerf power measurement configuration.
+Note that in contrast to SPECpower, the temperature sensor is not required, and
+the collection system is not coupled to the SUT and driving/controlling the SUT.
+All power delivered to the SUT must arrive through the power meter.
+
+DC-powered systems are also supported with an appropriate power meter, but may
+not be compared to AC-powered systems. If a wall-powered system is available in
+both an AC-powered and DC-powered configuration, we require using the
+AC-powered configuration.
+
+image::sut_power_setup.png[SUT Setup]
+
+== Wall-Powered Measurement Procedure
+
+. Connect an approved power meter from this list to the SUT.
+
+ For single enclosure devices (where all the components are
+ contained inside a single chassis), connect the power supply
+ to the meter. For disaggregated systems, or systems with
+ multiple nodes, all power supplies for the SUT components
+ must be connected to the power meter. For logging, the
+ power meter should be connected to an external computer
+ (not in the SUT) which is running the SPEC PTDaemon
+ logging software. See Figure 1 above.
+
+ This approach applies to both AC and DC powered systems,
+ but the type of power delivery must be specified and AC
+ and DC systems cannot be compared to one another due to
+ differences in conversion efficiency.
+
+. Synchronize system clock of two systems with Network Time Protocol (NTP)
+
+ More details to be shared as the procedure is being piloted.
```

Comment:
Can we clarify what this means for v0.5 submitters? Join the WG? Can we please elaborate with another statement or two?

Reply:
Added more details, and a link to NTP